JetNews/app/src/androidTest/java/com/example/jetnews/TestHelper.kt

```diff
@@ -61,14 +61,9 @@ fun ComposeTestRule.setMaterialContent(children: @Composable() () -> Unit) {
 }
 
 /**
- * Workarounds, these functions should be removed when UI testing improves
+ * Workaround, this function should be removed when UI testing improves
```

Comment:
Remove comment?

Reply:
Removed it and added a comment more relevant to the method